src/inc/quic_datapath.h

```diff
@@ -141,6 +141,21 @@ typedef struct CXPLAT_SEND_DATA CXPLAT_SEND_DATA;
 //
 typedef struct QUIC_BUFFER QUIC_BUFFER;
 
+typedef struct CXPLAT_INTERFACE {
```

Comment:
Do we really need to define this for all consumers of `quic_datapath.h`? Maybe just have `typedef struct CXPLAT_INTERFACE CXPLAT_INTERFACE` and move the full definition to the raw header.

Reply:
You are right. Just need a forward declaration here.